pom.xml

```diff
@@ -23,7 +23,7 @@
   <parent>
     <groupId>org.hawkular</groupId>
     <artifactId>hawkular-parent</artifactId>
-    <version>2</version>
+    <version>3-SNAPSHOT</version>
```

Comment:
Most likely, when we merge this, we'll want to change this version to "3" since that will probably be hawkular-parent's version